### PR TITLE
refactor: remove redudant workflows and removed serve package

### PR DESCRIPTION
**Pull request by EraKin575**
## Related Issue
  - Info about Issue or bug

Closes: #1430

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Review by PranshuSrivastava — CHANGES_REQUESTED** [review]

File: .github/workflows/test_workflow_scripts/golang-linux.sh [hunk 1]
```diff
@@ -71,19 +69,17 @@ sleep 5
 done
 
 # Start the gin-mongo app in test omde.
```
Comment (PranshuSrivastava) [inline .github/workflows/test_workflow_scripts/golang-linux.sh:71]: Please fix this typo as well

**Comment by Sarthak160** [comment]
@EraKin575 Java workflow is breaking, and since @PranshuSrivastava stated we should now use `java -jar "path-to-jar"` command instead of mvn run.

File: .github/workflows/test_workflow_scripts/python-linux.sh [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #! /bin/bash
 
-source ./../../../.github/workflows/test_workflow_scripts/test-iid.sh
```
Comment (PranshuSrivastava) [inline .github/workflows/test_workflow_scripts/python-linux.sh:3]: Please add this back as well.

**Review by PranshuSrivastava — CHANGES_REQUESTED** [review]

**Review by Sarthak160 — APPROVED** [review]
Ship it

**Review by PranshuSrivastava — APPROVED** [review]
LGTM

**Pull request merged**